lib/wallaby/query.ex

```diff
@@ -235,13 +236,17 @@ defmodule Wallaby.Query do
     Keyword.get(conditions, :count)
   end
 
+  def getNumber(%Query{conditions: conditions}) do
```

Comment:
please `get_number` although not sure that'ts the best name but I can't come up with a better atm